tensorboard/webapp/widgets/card_fob/card_fob_controller_component.ts

```diff
@@ -85,20 +90,15 @@ export class CardFobControllerComponent {
   }
 
   startDrag(fob: Fob, affordance: TimeSelectionAffordance) {
+    document.addEventListener('mousemove', this.mouseListener);
```

Comment:
I see RxJs `fromEvent()` used sometimes in the code base. Did you try using it? I don't know if it will make your life easier or not but it could be worth exploring. You can see an example usage here:

https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/widgets/histogram/histogram_component.ts;l=153;drc=c52d1385a5f4b0877b548bb398754adda50cf409

What you have here otherwise seems fine with me.